_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -121,7 +121,7 @@ Option | Required | Type | Description
 
 ## codec
 
-The `codec` determines how the `s3` source parses each S3 object.
+The `codec` determines how the `s3` source parses each S3 object. For increased and more efficient performance, you can use codecs in [combination]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/codec-processor-combinations/) with certain processors.
 
 ### newline codec
```

Comment:
```suggestion
### Newline codec
```

Reply:
See previous comments about headers in the Data Prepper section.